tests/e2e/utils/content-types.ts

```diff
@@ -372,6 +372,32 @@ export const fillAttribute = async (page: Page, attribute: AddAttribute, options
       const checkbox = page.getByRole('checkbox', { name: 'Required field' });
       await ensureCheckbox(checkbox, adv.required);
     }
+
+    if (isString(adv.regexp)) {
```

Comment:
Super duper minor but I don't like `adv` as a variable name. Maybe just
`advancedAttribute` ?

Reply:
Sure! it was just to shorten things, so how about the middle ground of 'advanced'?